electricpy/math.py

```diff
@@ -53,4 +53,196 @@ def step(t):
     to provide standard step-function as specified to
     be zero at x<0, and one at x>=0.
     """
-    return (_np.heaviside(t, 1))
\ No newline at end of file
+    return (_np.heaviside(t, 1))
+
+# Arbitrary Waveform RMS Calculating Function
+def funcrms(func, T):
```

Comment:
Just a Note:
I'm not entirely convinced that all of the RMS functions are truly best located in a `math` module. That said, I don't *know* where they should go. So they are fine here, for now. We may decide to move them again, later.